<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimedComputerManagementWidget</class>
 <widget class="QWidget" name="TimedComputerManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">TimedComputerManagementWidget</string>
  </property>
  <layout class="QGridLayout" name="pGridLayout_">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="pGroupBoxTimedComputerManagement_">
     <property name="title">
      <string>&amp;Timer Control</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="pLabelTime_">
          <property name="text">
           <string>&amp;Time (in minutes)</string>
          </property>
          <property name="buddy">
           <cstring>pComboBoxTimeout_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="pComboBoxTimeout_">
            <property name="insertPolicy">
             <enum>QComboBox::InsertAlphabetically</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="pGroupBoxComputerManagement_">
        <property name="title">
         <string>Computer Protection:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="pRadioButtonComputerManagementNone_">
           <property name="text">
            <string>&amp;None</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="pRadioButtonComputerManagementLock_">
           <property name="text">
            <string>&amp;Lock Computer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="pRadioButtonComputerManagementLogout_">
           <property name="text">
            <string>Logout &amp;User</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="pRadioButtonComputerManagementShutdown_">
           <property name="text">
            <string>Shutdown &amp;Computer</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
